.devcontainer/Dockerfile

```diff
@@ -43,16 +44,15 @@ ENV GO111MODULE=on
 # Install Go tools
 RUN \
     # --> Delve for debugging
-    go get github.com/go-delve/delve/cmd/dlv \
+    go install -v github.com/go-delve/delve/cmd/dlv@latest \
     # --> Go language server
-    && go get -v golang.org/x/tools/gopls \
+    && go install -v golang.org/x/tools/gopls@latest \
     # --> Goimports
-    && go get golang.org/x/tools/cmd/goimports \
-    # --> Gotestsum
-    && go get gotest.tools/gotestsum \
-    # --> Go symbols and outline for go to symbol support and test support 
-    && go get github.com/acroca/go-symbols@v0.1.1 && go get github.com/ramya-rao-a/go-outline@7182a932836a71948db4a81991a494751eccfe77 \
-    # --> Static checker
-    && go install honnef.co/go/tools/cmd/staticcheck \
-
-USER developer
\ No newline at end of file
+    && go install -v golang.org/x/tools/cmd/goimports@latest \
+    # # --> Gotestsum
+    && go install -v gotest.tools/gotestsum@latest \
+    # # --> Go symbols and outline for go to symbol support and test support 
+    && go install github.com/acroca/go-symbols@v0.1.1 \
+    && go install github.com/ramya-rao-a/go-outline@7182a932836a71948db4a81991a494751eccfe77 \
+    # # --> Static checker
+    && go install honnef.co/go/tools/cmd/staticcheck@latest
```

Comment:
And staticcheck that works with 1.16 is v2.2.1 (I think - check push.yml to be sure).

Reply:
it's 0.2.2